src/plugins/dashboard/public/application/embeddable/viewport/_print_viewport.scss

```diff
@@ -0,0 +1,9 @@
+.printViewport {
+  &__vis {
+    height: 600px; // These values might need to be passed in as dimensions for the report. I.e., print should use layout dimensions.
+    width: 975px;
+
+    // Some vertical space between vis, but center horizontally
+    margin: 10px auto;
+  }
+}
```

Comment:
See reporting config that might clash with these values: https://github.com/elastic/kibana/blob/65df961b5c45419ab490245f0e5a9809676e4f00/x-pack/plugins/reporting/server/config/schema.ts#L77-L80